redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -202,10 +202,9 @@ public static function updateMedia(array $data, ?string $userlogin = null): arra
 
         $return['ok'] = 1;
         $return['msg'] = rex_i18n::msg('pool_file_infos_updated');
-        $return['type'] = $data['file']['type'];
```

Comment:
wenn dieser index jetzt fehlt ist es ein BC break im EP?

Reply:
Den gab es in 5.12 auch nicht